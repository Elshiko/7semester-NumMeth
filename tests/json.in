[[15, 2, -1, -1, 22], [1, -10, -1, -2, -14], [2, 1, 12, 1, -10], [1, 1, 1, 11, -20]]
