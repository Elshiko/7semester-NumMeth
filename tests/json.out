[0.9999999999999999, 2.0, -0.9999999999999999, -2.0]